platform-sdk/swirlds-common/src/main/java/com/swirlds/common/threading/framework/internal/StoppableThreadImpl.java

```diff
@@ -575,11 +575,11 @@ private void close(final StopBehavior stopBehavior) throws InterruptedException
      */
     private void interruptClose() throws InterruptedException {
         join(joinWaitMs);
-
         if (isAlive()) {
             interrupt();
             joinInternal();
         }
+        doFinalCycleWork();
```

Comment:
`ReconnectNodeRemover.workQueue` does not set `finalCycleWork` so this change is a no-op for this particular case.

Reply:
QueueThreadImpl (used by ReconnectNodeRemover) does set `finalCycleWork` to `QueueThreadImpl.doFinalCycleWork()`, which flushes the remaining qeueu